qlkube/src/decorateOpenapi.js

```diff
@@ -2,14 +2,24 @@ const { apiGroups } = require('./apiGroups');
 
 const k8s = 'io.k8s.apimachinery';
 
+/**
+ *
+ * @param {*} oas
+ * This function filters the OpenApi made from k8s API.
+ * Starting from a list of paths,
+ * for each path into oas checks if at least one of our paths matches with it,
+ * so that paths that are never matched are removed.
+ * @returns
+ */
+
 function decorateOpenapi(oas) {
   let result = true;
   let keys = Object.keys(oas.paths);
   let removeKeys = keys.filter(key => {
     result = true;
     const paths = apiGroups.paths;
     paths.forEach(path => {
-      result &&= key.indexOf(path) == -1;
+      result = result && key.indexOf(path) === -1;
```

Comment:
```suggestion
      result = result && !key.includes(path);
```
I'd suggest something similar to increase readability, also below

Reply:
(to be tested clearly)